language.md

```diff
@@ -58,4 +58,6 @@ Ja runā latviski, [dodies šeit](./latvian/zefirs.md).
 
 Hvis du taler dansk, [tryk her](danish/velkommen.md)
 
-Ako govoris hrvatski, [klikni ovdje](croatian/sljez.md)
\ No newline at end of file
+Ako govoris hrvatski, [klikni ovdje](croatian/sljez.md)
+
+आपण मराठी बोलतो तर, येथे []क्लिक](marathi/song.md) करा 
```

Comment:
I believe the extra square brackets are preventing this from becoming a link correctly.